<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewEntry</class>
 <widget class="QDialog" name="AddNewEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>440</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>440</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Zrušit</string>
   </property>
  </widget>
  <widget class="QComboBox" name="monthComboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>54</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Měsíc</string>
   </property>
  </widget>
  <widget class="QComboBox" name="yearComboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>54</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Rok</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>381</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="userLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
